lib/granite/action/subject.rb

```diff
@@ -48,6 +52,10 @@ def initialize(*args)
 
       private
 
+      def find_subject_reflection
+        self.class.reflect_on_association(self.class._subject)
+      end
```

Comment:
This unrelated change can be undone. IMO, line 44 was inconsistent with the level of abstraction maintained in the rest of the `#initialize` method body.

Reply:
That's fine, but you have instance method than only calls 2 class methods. This should be a class method. Also let's rename this to `subject_reflection`.

Reply:
:+1: